handler/graphql.go

```diff
@@ -319,6 +320,16 @@ func WebsocketKeepAliveDuration(duration time.Duration) Option {
 	}
 }
 
+// WebsocketKeepAliveFirstInstantly allows you to reconfigure the keepalive behavior.
+// By default, keepalive is sent when first keep alive ping interval is over.
+// Set handler.connectionKeepAliveFirstInstantly = true to send first keepalive
+// instantly. This behaviour is required by a few frontend libraries.
+func WebsocketKeepAliveFirstInstantly(enabled bool) Option {
```

Comment:
Lets make this the default, looks like [the spec is pretty clear](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md#gql_connection_init)
> Client sends this message after plain websocket connection to start the communication with the server
> 
> The server will response only with GQL_CONNECTION_ACK + GQL_CONNECTION_KEEP_ALIVE (if used) or GQL_CONNECTION_ERROR to this message.
> 
> payload: Object : optional parameters that the client specifies in connectionParams